lib/ex_machina/ecto.ex

```diff
@@ -94,6 +121,25 @@ defmodule ExMachina.Ecto do
     |> drop_fields_with_nil_values
   end
 
+  @doc """
+  It is similar to `params_with_assocs/2`, but returned map keys are of type
```

Comment:
Typically there is a one sentence summary that goes as the first line. How about something like `Similar to `params_with_assocs/2` but converts keys to strings.`

And then the full description below that? Same would go for the other function